srcpkgs/libtd/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'libtd'
+pkgname=libtd
+version=1.6.6
+revision=1
+_commit=c78fbe4bc5e31395e08f916816704e8051f27296
+wrksrc="td-${_commit}"
+build_style=cmake
+build_helper=qemu
+hostmakedepends="gperf"
+makedepends="libressl-devel zlib-devel"
+short_desc="Cross-platform library for building Telegram clients"
+maintainer="Ivan Sokolov <ivan-p-sokolov@ya.ru>"
+license="BSL-1.0"
+homepage="https://core.telegram.org/tdlib"
+distfiles="https://github.com/tdlib/td/archive/${_commit}.tar.gz"
+checksum=adb84e39d95c3cde8a70d47410acd91af3953fc6ebbca152a1afbef9c7135e1e
+patch_args="-Np1"
+
+case "$XBPS_TARGET_MACHINE" in
```

Comment:
```sh
if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
	makedepends+=" libatomic-devel"
fi
```

Reply:
What are these systems? I also have to change [the patch](https://github.com/void-linux/void-packages/blob/9c749cf1e9f451a8f8d8f0be73d6d9351e6a7696/srcpkgs/libtd/patches/cross-compiling.patch#L103).

Reply:
Use https://github.com/void-linux/void-packages/blob/c4f6b779bcf998f462249c14946a29eecfa4be4f/srcpkgs/grpc/template#L27 instead of  patch fragment that links libatomic.

Reply:
> What are these systems? I also have to change [the patch](https://github.com/void-linux/void-packages/blob/9c749cf1e9f451a8f8d8f0be73d6d9351e6a7696/srcpkgs/libtd/patches/cross-compiling.patch#L103).

Using Chocimier's suggestion, patch is unnecessary, right?

armv5, armv6, mips, and ppc (32 bits) all missed it.
https://github.com/void-linux/void-packages/blob/c4f6b779bcf998f462249c14946a29eecfa4be4f/common/xbps-src/shutils/common.sh#L292-L302

Reply:
> Using Chocimier's suggestion, patch is unnecessary, right?

No, the patch is still needed to ignore `CMAKE_CROSSCOMPILING`.

Reply:
I was talking about your change for ATOMIC ;)

Reply:
It goes without saying. Thanks for your work.

Reply:
Already done